JenkinsFile

```diff
@@ -41,30 +46,30 @@ node {
       sh "mkdir -p .cf"
     }
 
-    def piazzaApiKey = getPiazzaApiKey(pcfSpace)
+    def piazzaApiKey = getPiazzaApiKey(pcfDomain)
 
     withEnv(["CF_HOME=.cf"]) {
       withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: "${env.PCF_CREDS}", usernameVariable: "CFUSER", passwordVariable: "CFPASS"]]) {
-        def iaBrokerUrl = "https://bf-ia-broker.$pcfDomain"
         sh """
           cf api ${env.PCF_API_ENDPOINT}
           cf auth ${CFUSER} ${CFPASS}
           cf target -o ${env.PCF_ORG} -s ${pcfSpace}
-          cf push ${deployAppName} -f manifest.jenkins.yml -p ${root}/${archiveName} --hostname ${deployAppName} -d ${pcfDomain} --no-start
-          cf set-env ${fullAppName} SPACE ${pcfSpace}
-          cf set-env ${fullAppName} DOMAIN ${pcfDomain}
-          cf set-env ${fullAppName} PIAZZA_API_KEY $piazzaApiKey
-          cf set-env ${fullAppName} GEOAXIS ${env.GEOAXIS_DOMAIN}
-          cf set-env ${fullAppName} GEOAXIS_CLIENT_ID ${GEOAXIS_CLIENT_ID}
-          cf set-env ${fullAppName} GEOAXIS_SECRET ${GEOAXIS_SECRET}
-          cf set-env ${fullAppName} GEOAXIS_SECRET ${GEOAXIS_SECRET}
-          cf set-env ${fullAppName} IA_BROKER ${iaBrokerUrl}
+          cf push ${deployAppName} -f manifest.jenkins.yml -p ${root}/${archiveName} --hostname ${deployAppName} -b ${env.JAVA_BUILDPACK_NAME} -d ${pcfDomain} --no-start
         """
         try {
-          sh "cf start ${deployAppName}"
+          sh """
+            cf set-env ${fullAppName} SPACE ${pcfSpace}
+            cf set-env ${fullAppName} DOMAIN ${pcfDomain}
+            cf set-env ${fullAppName} PIAZZA_API_KEY ${piazzaApiKey}
+            cf set-env ${fullAppName} GEOAXIS ${env.GEOAXIS_DOMAIN}
+            cf set-env ${fullAppName} GEOAXIS_CLIENT_ID ${GEOAXIS_CLIENT_ID}
+            cf set-env ${fullAppName} GEOAXIS_SECRET ${GEOAXIS_SECRET}
+            cf set-env ${fullAppName} GEOAXIS_SECRET ${GEOAXIS_SECRET}
```

Comment:
Why is GEOAXIS_SECRET in there twice? Not sure if it's supposed to be something else or just an extra copy.